versions.yml

```diff
@@ -197,18 +224,18 @@ modGroups:
       desc: Improves loading times when changing worlds
       url: https://github.com/kennytv/kennytvs-epic-force-close-loading-screen-mod-for-fabric/releases/download/1.1.1/forcecloseloadingscreen-1.1.1.jar
 
-    - name: ForgetMeChunk
-      license: No License
-      homepage: https://modrinth.com/mod/forgetmechunk
-      desc: Removes light-update when unloading chunks to fix lag spikes
-      url: https://github.com/mjwells2002/ForgetMeChunk/releases/download/1.0.4/forgetmechunk-1.0.4-1.18.X-1.19.X.jar
+    - name: Language Reload
+      license: MIT
+      desc: Reduces load times and adds fallbacks for languages
+      homepage: https://modrinth.com/mod/language-reload
+      url: https://cdn.modrinth.com/data/uLbm7CG6/versions/1.3.1/language-reload-1.3.1.jar
```

Comment:
seems kinda more like a misc thing? performance is moreso actual igname performance, not a reload time for changing a setting majority touch maybe ones